examples/justfile

```
@@ -261,8 +282,17 @@ ipfs-docker-start: _check-docker
     # Pull latest kubo image if not present
     docker pull ipfs/kubo:latest
     
+    # Determine network mode based on OS
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        # macOS - use port mapping
+        NETWORK_ARGS="-p 4001:4001 -p 8080:8080 -p 5001:5001"
+    else
+        # Linux (including CI) - use host networking for direct access
+        NETWORK_ARGS="--network host"
```

Why `--network host`? Aren't we missing ports for linux?

```suggestion
        NETWORK_ARGS="--network host -p 4001:4001 -p 8080:8080 -p 5001:5001"
```